### DISCUSSION TITLE
What do Search result web-part options allow items selection and allow multiple item selection allow us to do

### DISCUSSION THREAD

**Discussion by samg365** [microsoft-search/pnp-modern-search#2655] [Q&A] [answered]
### What do Search result web-part options allow items selection and allow multiple item selection allow us to do?

What sort of actions (OOTB or custom) do these settings allow us to do, does anyone have any examples they have used with this setting and is able to explain how to get them working?

Thanks in advance

**Comment by wobba** [comment, marked as answer]
See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/connections/item-selection/ (which could use more explanation). You can have one web part show items, and use values from those items as filters into another web part when selecting. Say you list a set categories in one web part, and upon clicking a category, use that as a filter in another web part. Sort of a more advanced/different kind of filter scenario.

Also see https://franckcornu.com/post/modern-search-focus-item-selection/

**Reply by samg365** [reply]
Thanks @wobba that has cleared it up for me.
